homeassistant/components/lg_soundbar/media_player.py

```diff
@@ -55,7 +57,7 @@ def __init__(self, discovery_info):
         self._treble = 0
 
         self._device = temescal.temescal(host, port=port, callback=self.handle_event)
```

Comment:
The reason for the stack trace is here. We're register ming the callback already here before the entity has been added to home assistant.

Preferably the library should be updated to allow registering the callback after creating the device instance.

A workaround is to use our dispatch helper to signal the entity callback method from the registered device callback function. Connect the signal in entity coroutine method `async_added_to_hass`.

Reply:
Here's the relevant code in the library:
https://github.com/google/python-temescal/blob/ebc83f93e5ab35b83a0443e8bb8af973c0c2c32a/temescal/__init__.py#L79-L81